src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs

```diff
@@ -297,9 +293,19 @@ private void RunProcessingLoop()
             if (_logger.IsInfo) _logger.Info("Block processor queue stopped.");
         }
 
+        private void FireProcessingQueueEmpty()
+        {
+            if (IsEmpty)
+            {
+                ProcessingQueueEmpty?.Invoke(this, EventArgs.Empty);
+            }
+        }
+
         public event EventHandler ProcessingQueueEmpty;
 
         public bool IsEmpty => _blockQueue.Count == 0 && _recoveryQueue.Count == 0;
+        
+        public int Count => _blockQueue.Count + _recoveryQueue.Count;
```

Comment:
strange public property on BlockchainProcessor -> Count

Reply:
or is it ProcessingQueue? then ok

Reply:
Made it explicit